x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/inference/TextEmbedding.java

```diff
@@ -40,23 +40,32 @@ public class TextEmbedding extends InferenceFunction<TextEmbedding> {
 
     @FunctionInfo(
         returnType = "dense_vector",
-        description = "Generates dense vector embeddings for text using a specified inference endpoint.",
-        appliesTo = { @FunctionAppliesTo(lifeCycle = FunctionAppliesToLifecycle.DEVELOPMENT) },
+        description = "Converts a constant text into dense vector embeddings using a inference endpoint. "
```

Comment:
not sure about the term `constant text` tbh, I'd say something like "Converts a string of text"

Reply:
I used your suggestion instead of my original one.